package.json

```diff
@@ -1,6 +1,26 @@
 {
   "name": "homeday-blocks",
   "version": "2.5.0",
+  "author": "Homeday - Frontend Team",
+  "description": "A Vue component library built by Homeday's frontend team.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/homeday-de/homeday-blocks.git"
```

Comment:
Don't need `git+` here. Can be just `github:homeday-de/homeday-blocks` or the full GitHub URL.

Reply:
Thanks, I'll update to full github url.

https://docs.npmjs.com/files/package.json#repository

(I'm adding links for my own reference ;) )